src/constants.ts

```diff
@@ -39,6 +39,7 @@ export const defaultOptions: Options = {
   eventsUrl: EVENTS_URL,
   pollInterval: PULL_INTERVAL,
   eventsSyncInterval: EVENTS_SYNC_INTERVAL,
+  axiosTimeout: 30000,
```

Comment:
Can we extract this to a declared constant like the other intervals?

Reply:
Done: https://github.com/harness/ff-nodejs-server-sdk/commit/cb549e33d6acf0d7923bf71ec940134f36477ffd